examples/api/smtlib/quickstart.smt2

```diff
@@ -2,20 +2,38 @@
 (set-option :produce-models true)
 (set-option :produce-unsat-cores true)
 (set-option :incremental true)
+; necessary to print in the unsat core assertions that do not have names
+(set-option :dump-unsat-cores-full true)
+
+; Declare real constanst x,y
 (declare-const x Real)
 (declare-const y Real)
+
+; Our constraints regarding x and y will be:
+;
+;   (1)  0 < x
+;   (2)  0 < y
+;   (3)  x + y < 1
+;   (4)  x <= y
+
 (assert (< 0 x))
 (assert (< 0 y))
 (assert (< (+ x y) 1))
-(assert (<= x y))
+(assert (<= 0 x y))
```

Comment:
I would rather keep this line as it was so that it matches (4).

Reply:
This was a mistake. I fixed now.